docs/cosmology/index.rst

```diff
@@ -4,16 +4,24 @@
 Cosmological Calculations (`astropy.cosmology`)
 ***********************************************
 
+.. toctree::
```

Comment:
>  just putting the I/O docs into the main Using astropy.cosmology section. The existing docs there are that long, and the I/O is not too long either.

I think I prefer to separate out the details. I'll experiment with having a short subsection and pointing to the more detailed docs. In particular, I want separate docs for when I/O methods are added for latex, html, `CLASS` and `CAMB` ini, and `Emcee` and `MontePython` etc MCMC outputs.

> But in any case, you really want the I/O docs to show up in the TOC, otherwise they are basically lost. I specifically went looking for them for review and couldn't find it until I went to the PR. Many/most people won't read the intro.

Good point. I'll definitely make sure it shows up in the TOC and is mentioned in a few places.